include/deal.II/fe/fe_interface_values.h

```diff
@@ -553,6 +575,17 @@ FEInterfaceValues<dim, spacedim>::reinit(const CellIteratorType &cell,
 
 
 
+template <int dim, int spacedim>
+inline double
```

Comment:
We don't have `inline` for the other functions here, but it doesn't really matter.

Reply:
That too is just copied from `fe_values.h`. If you want me to, I'll commit to making all of the functions `inline` in a follow-up patch, but for the moment I'm eager to get this patch merged because I have an external project that relies on it.

Reply:
No, it's fine with me.